example/text_classification/lib/classifier.dart

```diff
@@ -43,7 +43,10 @@ class Classifier {
 
   void _loadModel() async {
     // Creating the interpreter using Interpreter.fromAsset
-    _interpreter = await Interpreter.fromAsset(_modelFile);
+    final gpuDelegateV2 = GpuDelegateV2();
+    final options = InterpreterOptions()..addDelegate(gpuDelegateV2);
```

Comment:
Typo on the double dots?

Reply:
[Cascade notation](https://news.dartlang.org/2012/02/method-cascades-in-dart-posted-by-gilad.html), compiles as expected

Reply:
FYI: I'll undo the changes in this file once the GPU delegates are working. Keeping this around so it's easy to reproduce the issue

Reply:
TIL - Thanks! Still learning Dart (I've been doing Android for 12 years) since I think Flutter has a lot of advantages, but also a little behind where I'd like to be :)